src/Resources/views/CRUD/list_outer_rows_mosaic.html.twig

```diff
@@ -30,7 +30,7 @@ This template can be customized to match your needs. You should only extends the
 
                             <div class="mosaic-inner-box-default">
                                 {% block sonata_mosaic_background %}
-                                    <img src="{{ meta.isImageAvailable is defined and meta.isImageAvailable ?
+                                    <img src="{{ meta.isImageAvailable is not defined or (meta.isImageAvailable is defined and meta.isImageAvailable) ?
```

Comment:
Would this work?

```suggestion
                                    <img src="{{ meta.isImageAvailable|default(true) ?
```

Reply:
@greg0ire I think better to do:
```
<img src="{{ meta.isImageAvailable is defined and not meta.isImageAvailable ?
                                        sonata_admin.getMosaicBackground() :
                                        meta.image }}" alt="{{ meta.title }}" />
```
To keep default background config.
But isn't it even better to use `Metadata` class from admin bundle instead of the one from block bundle?
Forbidden?

Reply:
In any case this fix should be merged...

Reply:
@greg0ire @core23 @OskarStark What is your opinion about use of `Metadata` class from admin bundle in `MediaAdmin`? Should I create another PR for this?

Reply:
Isn't `meta.isImageAvailable is defined and not meta.isImageAvailable` strictly equivalent to `not meta.isImageAvailable|default(true)`? If yes we could simply use `meta.isImageAvailable|default(true) ? meta.image : sonata_admin.getMosaicBackground()`

Reply:
@greg0ire that won't work.
when `false` is returned by `meta.isImageAvailable` it's also considered as empty value by `default` filter (see `_twig_default_filter` implementation).